src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/TalismanListener.java

```diff
@@ -223,14 +223,19 @@ public void onKill(EntityDeathEvent e) {
     public void onItemBreak(PlayerItemBreakEvent e) {
         if (Talisman.trigger(e, SlimefunItems.TALISMAN_ANVIL)) {
             PlayerInventory inv = e.getPlayer().getInventory();
+
             int slot = inv.getHeldItemSlot();
 
             // Did the tool in our hand break or was it an armor piece?
             if (!e.getBrokenItem().equals(inv.getItemInMainHand())) {
-                for (int s : armorSlots) {
-                    if (e.getBrokenItem().equals(inv.getItem(s))) {
-                        slot = s;
-                        break;
+                if (e.getBrokenItem().equals(inv.getItemInOffHand())) {
+                    slot = 40;
+                } else {
+                    for (int s : armorSlots) {
+                        if (e.getBrokenItem().equals(inv.getItem(s))) {
+                            slot = s;
+                            break;
+                        }
                     }
                 }
```

Comment:
`e.getBrokenItem()` is used so often, it would make sense to create a variable for it (`brokenItem`)